x-pack/plugins/fleet/public/applications/integrations/sections/epm/components/package_list_grid.tsx

```diff
@@ -205,7 +205,7 @@ function GridColumn({ list, showMissingIntegrationMessage = false }: GridColumnP
               ) : (
                 <FormattedMessage
                   id="xpack.fleet.epmList.noPackagesFoundPlaceholder"
-                  defaultMessage="No packages found"
+                  defaultMessage="No integration found"
```

Comment:
Should this be "integrations" instead?

Reply:
> Should this be "integrations" instead?

@kpollich yeah I wasn't sure if "No integration" is correct, I think you should be right since you are a native speaker :)